packages/devextreme/js/__internal/grids/grid_core/sticky_columns/m_sticky_columns_utils.ts

```diff
@@ -0,0 +1,39 @@
+import { StickyPosition } from './const';
```

Comment:
minor, at your discretion
I would name this file just as 'utils.ts', not 'm_sticky_columns_utils.ts'